src/providers/node/turborepo.rs

```diff
@@ -0,0 +1,44 @@
+use serde_json::Value;
+use std::collections::HashMap;
+
+use anyhow::Result;
+use serde::{Deserialize, Serialize};
+
+use crate::nixpacks::{app::App, environment::Environment};
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct TurboJson {
+    pub pipeline: HashMap<String, Value>,
+}
+
+pub struct Turborepo;
+
+impl Turborepo {
+    pub fn is_turborepo(app: &App) -> bool {
```

Comment:
Thoughts on having a way to skip turbo detection? Maybe something like `NIXPACKS_SKIP_TURBO`

Reply:
Turborepo configuration files are pretty unique.. useless IMO.